server/api/email.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+const Boom = require('boom');
+const Proposal = require('../models/proposal');
+const Feedback = require('../models/feedback');
+const User = require('../models/user');
+const Mailer = require('../mailer');
+const Config = require('../../config');
+var addDays = require('date-fns/addDays');
+var format = require('date-fns/format');
+
+const register = function (server, options) {  
+
+  server.route({
+    method: 'POST',
+    path: '/api/email/',
+    options: {
+      auth: {
+        strategies: ['simple', 'session'],
+        scope: ['reviewer', 'root', 'coordinator', 'chair']
+      }      
+    },      
+    handler: async function (request, h) {
+    
+      let template = request.payload.templateName;
+      let subject;  
+      let emailOptions;         
+      let emailTemplateData;
+      
+      /*
+        Case for when Chair assigns reviewers to a proposal. Emails are sent to all selected reviewers
+      */
+        if(template === 'reviewers-to-review-proposal'){
+            subject = 'You have been assigned to review a proposal';
+            const proposalDoc = await Proposal.findById(request.payload.proposalId); 
```

Comment:
Move this line to out of if statements so you don't have to repeat this inside all if blocks. Also do a check to make  sure proposal is not null or undefined, if it is, throw Boom.notFound("document not found")